bindings/python/iota_sdk/types/mana.py

```diff
@@ -20,3 +20,25 @@ class ManaAllotment:
     mana: int = field(metadata=config(
         encoder=str
     ))
+
+
+@json
+@dataclass
+class ManaRewardsResponse:
```

Comment:
probably fine to do it here

Reply:
For this one it might make sense, but for many like BlockMetadataResponse, ManaRewardsResponse, OutputWithMetadataResponse, the Response suffix looks pretty useless :shrug:

Reply:
This specific comment is just about moving them together though, you think we shouldn't?

Reply:
Somehow commented on the wrong comment, I'm fine moving all the response types in a single place

Reply:
moved them